### PR TITLE
fixed bug 1193, setted up correct size for button 'sing up'

### PR DESCRIPTION
**Pull request by skrebetsnataliia**
Setted up correct size (91px x 32px) for button 'sing up' in 'header component'

Previous:
image

Result:
image

https://github.com/ita-social-projects/GreenCity/issues/1193

### CODE REVIEW

**Comment by vitaliipolitylo** [comment]
Checks failed because of style linter problems.
 313:13  ✖  Expected single space after ":" with a single-line declaration   declaration-colon-space-after
Run 'style linter' to check for style issues.

**Review by yulkats — APPROVED** [review]

**Review by luchward — APPROVED** [review]

**Review by Benerous — APPROVED** [review]

**Review by Benerous — APPROVED** [review]

**Review by vitaliipolitylo — APPROVED** [review]

**Review by HryshkoAnton — APPROVED** [review]

File: src/app/component/layout/components/header/header.component.scss [hunk 1]
```diff
@@ -169,7 +169,7 @@ $screen-320: " screen and (max-width : 320px)";
   cursor: pointer;
   width: auto;
   min-width: 138px;
-  padding: 0 4px 0;
+  padding: 15px 34px;
   height: 48px;
```
Comment (mr-pavlenko) [inline src/app/component/layout/components/header/header.component.scss:173]: Why these paddings are different?

File: src/app/component/layout/components/header/header.component.scss [hunk 2]
```diff
@@ -304,12 +304,13 @@ $screen-320: " screen and (max-width : 320px)";
       width: auto;
       min-width: 86px;
       margin-left: 25px;
-      padding: 0 4px 0;
+      padding: 17px 33px;
```
Comment (mr-pavlenko) [inline src/app/component/layout/components/header/header.component.scss:307]: the same

**Review by mr-pavlenko — CHANGES_REQUESTED** [review]
add screenshots

File: src/app/component/layout/components/header/header.component.scss (hunk 1 above)
Comment (skrebetsnataliia) [reply src/app/component/layout/components/header/header.component.scss:173]: I fixed paddings according to mockup.
image

File: src/app/component/layout/components/header/header.component.scss (hunk 2 above)
Comment (skrebetsnataliia) [reply src/app/component/layout/components/header/header.component.scss]: I have fixed this.

**Review by luchward — APPROVED** [review]

**Pull request merged**